ddpui/api/pipeline_api.py

```
@@ -850,6 +850,28 @@ def cancel_queued_manual_job(request, flow_run_id, payload: TaskStateSchema):
+                logger.info(f"Cancelling Airbyte jobs for flow_run_id {flow_run_id}. Connections: {connection_ids}")
+                for conn_id in connection_ids:
+                    try:
+                        # Fetch and cancel running jobs for this connection
+                        job_types = ["sync", "reset_connection", "clear", "discover_schema", "refresh"]
+                        jobs_res = airbyte_service.get_jobs_for_connection(conn_id, job_types=job_types, limit=10)
+                        for job_entry in jobs_res.get("jobs", []):
+                            if job_entry["job"]["status"] == "running":
+                                job_id = job_entry["job"]["id"]
+                                airbyte_service.cancel_job(job_id)
+                                logger.info(f"Cancelled Airbyte job {job_id} for connection {conn_id}")
+                    except Exception as ab_error:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)